src/Nancy.Authentication.Stateless/StatelessAuthentication.cs

```diff
@@ -74,16 +74,10 @@ public static void Enable(INancyModule module, StatelessAuthenticationConfigurat
 
             return context =>
             {
-                try
-                {
-                    context.CurrentUser = configuration.GetUserIdentity(context);
-                    return context.Response;
-                }
-                catch (Exception)
-                {
-                    return context.Response;
-                }
+                context.CurrentUser = configuration.GetUserIdentity(context);
+
+                return null;

```

Comment:
BTW, this is what the other auths do as well:
 - [Basic](https://github.com/NancyFx/Nancy/blob/665a59a23ce3df74672f3334bf653947151f11a3/src/Nancy.Authentication.Basic/BasicAuthentication.cs#L76)
 - [Forms](https://github.com/NancyFx/Nancy/blob/665a59a23ce3df74672f3334bf653947151f11a3/src/Nancy.Authentication.Forms/FormsAuthentication.cs#L211)
 - [Token](https://github.com/NancyFx/Nancy/blob/d2d0240327efe8875025c4a3c68c093ac9bfb877/src/Nancy.Authentication.Token/TokenAuthentication.cs#L71)